### ISSUE TITLE
Stuck on Fingerprint Unlock

### ISSUE THREAD

**Issue by DaAwesomeP** [PhilippC/keepass2android#1423]
If Keepass2Android is prompting for a fingerprint and I lock my screen, then I get into a deadlock where I must restart my phone.

The app loses track of the unlock prompt, which is drawn over all apps. So it just sits there permanently and the cancel and soft back buttons don't work. I can only navigate my phone using the soft nav buttons. If I use the app switch button to get back to Keepass, then it prompt again for a fingerprint and draws over the existing prompt. This one works but when it goes away the broken one behind it lingers. It is even drawn over the lock screen.

**Comment by ahkeepass** [comment]
Yes, big problem for me as well. I turned off this feature to be able to use my phone.

**Comment by sba923** [comment]
I confirm I get this quite often with 1.08c-r1

**Comment by PhilippC** [comment]
@sba923 can you provide a log to crocoapps@gmail.com please?

**Comment by sba923** [comment]
Already done, on Nov15 at 11:58 CET

**Comment by PhilippC** [comment]
hmm, I don't see anythign in my inbox? Can you please send again?

**Comment by sba923** [comment]
Done. I'm sending this via a Spamgourmet.com-supplied "single-use address" ending with domain "xoxy.net."

This time I used two different addresses, one of the copies should make it at least to your "spam" folder.

**Comment by sba923** [comment]
I now have 1.08d-r0 (online) running on my phone. Does this include the fix for this issue?

**Comment by 6str** [comment]
i have this after my screen is unlocked on an LG G8s android 10

**Comment by sba923** [comment]
I hate to say, but: the problem _had_ been fixed, but came back recently. Running 1.09a-pre1 on Android 10 Nokia 7.2

**Comment by 6str** [comment]
yeah. i had it this morning too. had to restart my phone

**Comment by RichardNobel** [comment]
This issue also occurs on my Nokia 6 phone (running Android 9, with Microsoft Launcher). 
The fingerprint unlock screen of keepass2android covers the system lockscreen. Neither will respond to the fingerprint sensor. So that's a deadlock situation, like @DaAwesomeP mentioned in the very first comment in this topic.
